sceptre/aws-opendata/config/prod/cfn-s3objects-macro.yaml

```diff
@@ -0,0 +1,6 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/cfn-s3objects-macro/0.0.5/cfn-s3objects-macro.yaml
```

Comment:
The most recent version of this lambda is `0.0.6`, and it looks like it was a security update

Reply:
updated